TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -19,10 +19,10 @@ public class RoutingManager
         : AbstractGeometryObservingManager,
           IRoutingManager
     {
-        private RoutingManager() { }
-
         public static readonly RoutingManager Instance = new RoutingManager();
 
+        private RoutingManager() { }
+
         private const NetInfo.LaneType ROUTED_LANE_TYPES =
```

Comment:
what about this one?
![image](https://user-images.githubusercontent.com/26344691/159155328-d6c32bc9-8cd2-4a33-8202-c7e54eba73fa.png)

Reply:
why is this warning important?
![image](https://user-images.githubusercontent.com/26344691/159155337-17d66eaa-5ab7-44c3-9687-321340263cef.png)

Reply:
this warning seems unnecessary:
![image](https://user-images.githubusercontent.com/26344691/159155342-9165692f-d5b8-4c21-90c3-a6293bde5e94.png)